### ISSUE TITLE
Vegan advertising (follow up)

### ISSUE THREAD

**Issue by huggi** [guaka/veganwiki#22]
Hi Guaka

I've requested access to have google-ads and invited veganwikimasters@gmail.com to join the program, here's the code. Not sure how that can be entered in the wiki. Any idea?
I think it would make sense having more dynamic ad integration on the wiki itself.

Let me know what you think and we can try it for a while. Also curious to learn what sites are accessed by real users within the last 24 hours as I'm adding constantly new articles and it would be good to see if someone reads them :)

**Comment by guaka** [comment]
Hmm I didn't want adsense because it will also advertise non vegan products
and places. It's possible to block ads from showing though. So if you're
going to check ads every week this can work. We can also add a prominent
link to contact us if there are non vegan ads.

sent from my mobile phone

**Comment by guaka** [comment]
Another complication is upcoming EU cookie consent required by Adsense, see also http://webmasters.stackexchange.com/q/84847/24487

**Comment by guaka** [comment]
To see how it could work, check http://deletionpedia.org/en/Main_Page
Also note that I feel much less constrained putting ads on Deletionpedia, as it's not a community project. Ad insert code can be easily adapted, and in fact it's already used for veganwiki.info's veganic ads.

**Comment by huggi** [comment]
Did not know about the EU cookie thing, how about Switzerland? I have quite good contacts in this field, I'll guess it's not relevant for this non-EU country, but moving all the stuff because of that would make really no sense. I like the Deletionpedia with the banners. 

I think we should try out different things if we have options, I'm not an expert yet in terms of "online advertisment" as long it's not so annoying like the HC website everything is okay for me :)

**Comment by guaka** [comment]
It's about who is looking at the ad, not from where you serve it. (So CH is not going to help here.)

**Comment by huggi** [comment]
Easy to cheat the law, buy a proxy in a non-eu country and redirect all your traffic there, so you don't have to move your content (wherever it is).

**Comment by guaka** [comment]
It's not about cheating the law. A lot of govt websites are cheating the
law inside the EU.

It's about Google AdSense requiring user consent. And you can't cheat
Google - not for long at least.

**Comment by simison** [comment]
@guaka btw did you have a source where Google is stating that requirement? (No worries if it's not at hands, I'll google (har har) it up later)

**Comment by guaka** [comment]
http://adsense.blogspot.co.uk/2015/07/introducing-new-user-consent-policy.html

**Comment by huggi** [comment]
I don't believe in google, maybe abc.xyz (btw: what's up with http://www.veganwiki.xyz/?)

Just imagine, what you do with a private VPN to any city in the world, you can also do that with webservers at any CDN today, I know the CDN industry very well, with Cloudflare (I know Martin Levy, one of their strategist) within the top5. Google, Microsoft, Limelight, Akamai (where for e.g. whitehouse.gov is hosted in your nearest city, via your DNS query). 

The law is to static to follow what's already in practice, so being creative is always a grey area. 

your blog article vs. the law: https://en.wikipedia.org/wiki/Directive_on_Privacy_and_Electronic_Communications 

So said that, if you attract your users/viewers to a Swiss IP-address (not intellectual property, which is not in the european union), than you have to stick to the very strict swiss, not european law. 
Even if there is a court case against your company in the Netherlands, than they have to start a law enforcement treaty between the specific requesting country to the Swiss law. This is possible but you don't have to reveal your customer logfiles or anything. In terms of privacy laws, that's very strict and a leading concept in terms of Consumer rights, which have more priority in Switzerland in the online and offline World than money (like in the rest of the world, as of what I have seen).

**Comment by simison** [comment]
@huggi: don't show the alert for users coming from EU —> Google will either revoke your adsense account or just won't show the ads for them. No way around it. It's not a matter of opinion or server hosting location.

**Comment by huggi** [comment]
Who needs Google?
